Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM openjdk:8u292-jre-slim
```

Comment:
Can you use the base image?
https://github.com/scalar-labs/scalar/blob/master/ledger/Dockerfile#L18

Reply:
Thank you for your suggestion.
Docker file updated with the above image.